### PR TITLE
Adds prepare-next-snapshot-version CI job

### PR DESCRIPTION
**Pull request by JayShortway**
As the title says.

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -14,6 +14,7 @@ REPO_NAME = "purchases-kmp"
 PATH_CHANGELOG_LATEST = './CHANGELOG-LATEST.md'
 PATH_CHANGELOG = './CHANGELOG.md'
 PATH_VERSIONS = './VERSIONS.md'
+FILE_WITH_SDK_VERSION = { './gradle/libs.versions.toml' => ["revenuecat-kmp = \"{x}\""] }
```
Comment (tonidero) [inline fastlane/Fastfile:17]: Nitpick, but since this is a map, I would make this plural: `FILES_WITH_SDK_VERSION`

**Review by tonidero — APPROVED** [review]
LGTM!

File: fastlane/Fastfile (hunk 1 above)
Comment (JayShortway) [reply fastlane/Fastfile:17]: But I was so happy it's just 1 file 😛 I hope we can keep that up as long as possible haha.

Comment (JayShortway) [reply fastlane/Fastfile]: Updated!

**Pull request merged**